pvlib/iotools/pvgis.py

```diff
@@ -510,6 +536,16 @@ def get_pvgis_tmy(latitude, longitude, outputformat='json', usehorizon=True,
     if map_variables:
         data = data.rename(columns=VARIABLE_MAP)
 
+    if roll_utc_offset is not None or coerce_year is not None:
+        # XXX: be explicit, test for identity not implicit booleaness
```

Comment:
what does these "xxx" mean?

Reply:
I use it as a warning to beware or pay attention or to flag problematic code. I used it here because I was worried about implicit booleaness, especially that `None` and 0 are both false, which could be a problem because zero is a valid UTC offset, but evaluates the same as `None` even though we want the opposite outcome.

Various places define it differently, but I disagree that's it's synonymous with or related to `FIXME:` or `TODO:`
- [wikipedia > computer programming comments > tags](https://en.wikipedia.org/wiki/Comment_(computer_programming)#Tags)
- [SO what does `XXX` mean?](https://softwareengineering.stackexchange.com/questions/65467/what-does-xxx-mean-in-a-comment)
- [SO what is the meaning of `XXX`?](https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments)
- [SO best practices rule 9](https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/)

Reply:
I removed the warning comments now that I handle `tz=None` in `_coerce_and_roll_tmy()` instead of here.